parity-deploy.sh

```diff
@@ -266,10 +266,10 @@ while [ "$1" != "" ]; do
         -n | --name )           shift
                                 CHAIN_NAME=$1
                                 ;;
-        -c | --chain )         shift
+        -c | --config )         shift
                                 CHAIN_ENGINE=$1
                                 ;;
-        -n | --nodes )    	    shift
+        -nn | --nodes )    	    shift
```

Comment:
single dash args should be single letter, I would remove the single letter `--name` alias